packages/ecc-client-ga4gh-tes/src/components/tes-service/definition/tesService.template.ts

```diff
@@ -0,0 +1,107 @@
+import { html, repeat, when } from '@microsoft/fast-element';
+
+const OtherTemplate: any = (x: any) => html`
+  <div class="template-container container key-value">
+    <div class="key">${x[0]}</div>
+    <div class="value">${x[1]}</div>
+  </div>
+`;
+
+const ArrayTemplate: any = (x: any) => html`
```

Comment:
I can see in the API that storage is the array. Can it have multiple values if so this implementation will break!

Reply:
I have the repeat directive for the array ie rendering all the variables, objects and strings of the array. So it shouldn't break. Or did I miss something here :sweat_smile: